runtime/tests/third_party/dummy_device.toml

```diff
@@ -1,4 +1,4 @@
-schema = 1
+schema = 2
 
 [device]
 name = "dummy.device.qubit"
```

Comment:
Erick suggested removing the device name, this might be a good opportunity to do it.

Reply:
Done.